changelogs/fragments/362-parted_documentation_fix.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
```

Comment:
I'm not sure about that: is there a changelogs key for documentation fixes?

Reply:
Doc fixes usually get no changelog fragment at all. You can simply remove it here.